### PR TITLE
Update README deployment process 

### PR DESCRIPTION
**Pull request by mckadesorensen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,25 @@ Cumulus.
...
+Set DAAC_DIR as an enviroment var:
 
-        $ make image
-        $ DAAC_DIR=$HOME/projects/acme-cumulus make container-shell
+      $ export DAAC_DIR=$HOME/projects/acme-cumulus
 
 Here DAAC_DIR is the absolute path to the fork of `CIRRUS-DAAC` that
 you would like to deploy.
 
+If DAAC_DIR is not set, make will give you the following error:
```
Comment (mckadesorensen) [inline README.md:42]: The root of this error is the addition of [force](https://github.com/asfadmin/CIRRUS-core/blob/3b0f19c29200bb951826d8da7db10e5405c9b23e/Makefile#L323)

File: Dockerfile [hunk 2]
```diff
@@ -21,6 +21,9 @@ RUN \
 
 # CLI utilities
 RUN yum install -y gcc gcc-c++ git make openssl unzip wget zip jq
+RUN git clone https://github.com/nasa/cumulus-dashboard.git && chmod 777 /cumulus-dashboard
```
Comment (mattp0) [inline Dockerfile:24]: I'd use 755 instead of 777 it's just probably not needed to give rwx for everyone just bad security hygiene

**Review by mattp0 — COMMENTED** [review]
Is the scope of this readme update include the other sections of the readme? The organization section looks like it could use some updates

File: Dockerfile (hunk 2 above)
Comment (mckadesorensen) [reply Dockerfile:24]: I did not mean to include that... Good catch.

Comment (mckadesorensen) [reply Dockerfile:24]: I am a bit confused how that got in here. That was me trying to figure out why I couldn't get the Dashboard working the other day.

**Comment by mckadesorensen** [comment]
> Is the scope of this readme update include the other sections of the readme? The organization section looks like it could use some updates

I was initially just trying to get the deployment process updated. OB DAAC is starting to use CIRRUS and I wanted the deployment instructions to be accurate.

**Comment by mckadesorensen** [comment]
I could look at updating the other sections.

**Review by mattp0 — APPROVED** [review]
I think this good doesn't effect any current commands but adds one for boot strapping a new deployment

**Comment by mikedorfman** [comment]
Looks good to me other than my one nit-picky comment!  Agreed with @mattp0 - the README could use some love at some point, but probably out of scope of this change.

**Pull request merged**